apps/api/src/applicant/external-api.service.ts

```diff
@@ -533,12 +533,11 @@ export class ExternalAPIService {
    * @returns object that use in upsert milestone/status
    */
   async mapMilestones(data: any, mappedApplicantList: string[]) {
-    const { users } = await this.getApplicantMasterData();
-    const savedApplicants = await this.ienapplicantRepository.findByIds(mappedApplicantList);
```

Comment:
we don't need this anymore?  If not, cant we check `if (!mappedApplicantList.length)` prior to calling this function?

Reply:
mappedApplicantList already contains the upserted applicant IDs and `findByIds` returns nothing. I guess it's because of `transaction`.